docs/tutorials/podman_tutorial.md

```diff
@@ -123,13 +123,14 @@ sudo make install PREFIX=/usr
 This sample container will run a very basic httpd server that serves only its index
 page.
 ```console
-podman run -dt -e HTTPD_VAR_RUN=/var/run/httpd -e HTTPD_MAIN_CONF_D_PATH=/etc/httpd/conf.d \
+podman run -dt -p 8080:8080/tcp -e HTTPD_VAR_RUN=/var/run/httpd -e HTTPD_MAIN_CONF_D_PATH=/etc/httpd/conf.d \
                   -e HTTPD_MAIN_CONF_PATH=/etc/httpd/conf \
                   -e HTTPD_CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/httpd/ \
                   registry.fedoraproject.org/f27/httpd /usr/bin/run-httpd
 ```
```

Comment:
This command worked for me as root, but failed as rootless:

```
$ podman run -dt -p 8080:8080/tcp -e HTTPD_VAR_RUN=/var/run/httpd -e HTTPD_MAIN_CONF_D_PATH=/etc/httpd/conf.d \
>                   -e HTTPD_MAIN_CONF_PATH=/etc/httpd/conf \
>                   -e HTTPD_CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/httpd/ \
>                   registry.fedoraproject.org/f27/httpd /usr/bin/run-httpd
Trying to pull registry.fedoraproject.org/f27/httpd...Getting image source signatures
Copying blob ff3dab903f92 [==================================] 80.7MiB / 80.7MiB
Copying blob 9347d6e9d864 [====================================] 7.3MiB / 7.3MiB
Copying blob 2fc5c44251d4 [==================================] 44.8MiB / 44.8MiB
Copying config 18f01f6f77 [====================================] 6.6KiB / 6.6KiB
Writing manifest to image destination
Storing signatures
Error: error from slirp4netns while setting up port redirection: map[desc:bad request: add_hostfwd: slirp_add_hostfwd failed]
```

@giuseppe do we have a slirp4netns dependency that we need to document in here?  I'm running podman 1.1.2

Reply:
we need slirp4netns 1.13

Reply:
That didn't come directly in with the 1.1.2 kit,  will it with 1.2?  Do we need to add anything here in the the tutorial?

Reply:
Hmmm... As I started out I just wanted to fix one issue as I ran into a brick wall by just trying out this tutorial. But I can add a hint that this rootless command needs at least slirp4netns v0.3.0.